Dockerfile

```diff
@@ -1,25 +1,33 @@
-FROM node:12-alpine AS build
+FROM node:14-alpine AS build
 
-# Install build dependencies for node modules
-RUN apk add git python make g++
+WORKDIR /app
+
+RUN apk add --no-cache --virtual .build-deps git python make automake autoconf g++ libpng-dev libtool nasm file
+
+COPY package.json yarn.lock ./
+
+RUN yarn install --frozen-lockfile
+
+COPY . .
+
+# Build SSR app for production in standalone mode
+
+RUN yarn build:ssr --production --standalone
+
+# Build final image
+FROM node:14-alpine
 
-# Set workdir
 WORKDIR /app
 
-# Add package.json and yarn.lock
-ADD package.json yarn.lock ./
+COPY .docker/package.json .docker/nuxt.config.js ./
 
-# Install dependencies
-RUN yarn install
+# Copy client files from the build image
+COPY --from=build /app/.nuxt ./.nuxt
+COPY --from=build /app/static ./static
```

Comment:
Not sure about the requirements of `./` before the dests (I'm just nitpicking)

Reply:
The linter in VSCode gave me shit if I didn't use them :yum:

Reply:
Personally I would use absolute target paths (so its clear at first sight where things end up), however this is just IMHO